### PR TITLE
Webpushr Trace added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://radioink.com/
https://www.healthy-holistic-living.com/
https://www.heartlandvetsupply.com/
https://radiofacts.com/
https://unrealfacts.com/
http://www.dailycameranews.com/
https://www.desiblitz.com/

### CODE REVIEW

File: src/technologies/w.json [hunk 1]
```diff
@@ -843,6 +843,18 @@
...
+    "cats": [
+      32
+    ],
+    "description": "Webpushr is the new standard for Web Push Notifications.",
+    "icon": "webpushr.svg",
+    "js": {
+      "webpushr": ""
+    },
+    "scripts": "cdn\\.webpushr\\.com/app\\.min\\.js",
+    "website": "https://www.webpushr.com"
```
Comment (nurbek91) [inline src/technologies/w.json:856]:
```
"pricing": [
      "recurring",
      "freemium",
      "low"
    ],
```

File: src/technologies/w.json [hunk 2]
```diff
@@ -843,6 +843,18 @@
     },
     "website": "https://webpack.js.org/"
   },
+  "Webpushr": {
+    "cats": [
+      32
+    ],
+    "description": "Webpushr is the new standard for Web Push Notifications.",
```
Comment (nurbek91) [inline src/technologies/w.json:850]: Less promotional description with structure like 
Webpushr is .........................

File: src/technologies/w.json [hunk 3]
```diff
@@ -843,6 +843,18 @@
...
+    "icon": "webpushr.svg",
+    "js": {
+      "webpushr": ""
+    },
```
Comment (nurbek91) [inline src/technologies/w.json:854]: "js": {
      "WebPushr.notificationCard": "",
      "webpushr_display_button": ""
    },

File: src/technologies/w.json [hunk 4]
```diff
@@ -843,6 +843,18 @@
...
+    "scripts": "cdn\\.webpushr\\.com/app\\.min\\.js",
```
Comment (nurbek91) [inline src/technologies/w.json:855]: You can delete or leave if there are examples without js objects and this code will work

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR. Need improvements

File: src/technologies/w.json [hunk 5]
```diff
@@ -843,6 +843,25 @@
     },
     "website": "https://webpack.js.org/"
   },
+  "Webpushr": {
+    "cats": [
+      32
+    ],
+    "description": "Webpushr is Push Notification which supports both Mobile & Desktop devices.",
```
Comment (nurbek91) [inline src/technologies/w.json:850]: `"description": "Webpushr is a web push notification platform that supports mobile and desktop devices."
`

File: src/technologies/w.json [hunk 6]
```diff
@@ -843,6 +843,25 @@
...
+    "icon": "webpushr.svg",
+    "js": {
+      "WebPushr.notificationCard": "",
+      "webpushr": "",
```
Comment (nurbek91) [inline src/technologies/w.json:854]: can be deleted
`"webpushr": "",  `
If it is possible to check like this, then it is better to add it in this way to allow us to minimize random triggers
`"WebPushr.notificationCard": "",`

File: src/technologies/w.json [hunk 7]
```diff
@@ -843,6 +843,25 @@
...
+      "webpushr_display_button": ""
+    },
+    "pricing": [
+      "recurring",
+      "freemium",
+      "low"
+    ],
+    "scripts": "cdn\\.webpushr\\.com/app\\.min\\.js",
```
Comment (nurbek91) [inline src/technologies/w.json:862]: This line can be deleted If we want to leave a link where there are no js objects, this line will help us identify the technology

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/w.json (hunk 7 above)
Comment (dhavalveera) [reply src/technologies/w.json:862]: I think we can put this as well, where there are no js objects and mostly this line will help us to identify the technology.

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**